symfony/framework-bundle/3.3/Makefile

```diff
@@ -16,7 +16,7 @@ cache-warmup: cache-clear
 ifdef CONSOLE
 	@$(CONSOLE) cache:warmup
 else
-	@printf "cannot warmup the cache (needs symfony/console)\n"
+	@printf "Cannot warmup the cache (needs symfony/console)\n"
```

Comment:
If I'm right, "warmup" word doesn't exist. It's `warm-up` as a noun and `warm up` as a verb, so in this case it should be "Cannot warm up the cache ..."

Reply:
👍  for "warm up"

Reply:
@bocharsky-bw Can you make the change?

Reply:
Done